ci/request_pr_exp.py

```diff
@@ -110,11 +110,6 @@ def _parse_args(cmd) -> argparse.Namespace:
       default=EXP_DELAY,
       help=('Delay each benchmark experiment by N seconds, default: '
             f'{EXP_DELAY}.'))
-  parser.add_argument(
```

Comment:
will this break invocations for people passing `-f` out of habit?

Reply:
Yep, it would. Let me notify everyone.
@DavidKorczynski @arthurscchan: We no longer need to add `-f` to `gcbrun` commands, old bucket/job will be deleted automatically when the new one is requested.